### ISSUE TITLE
Abandoned it appears?

### ISSUE THREAD

**Issue by smooreace** [AstronIsTaken/du-craft#19]
Its unfortunate. This was the only thing out there.

**Comment by AstronIsTaken** [comment]
Yeah, unfortunately I stopped playing DU after so called "full" release. Game has too many issues and too expensive for what it offers. And without playing I don't feel like updating the tool. 

Good thing is  domain du-craft.net payed off till 2025 or so and anyone is welcome to make code changes
